app/models/contact.rb

```diff
@@ -30,23 +30,22 @@ class Contact < ApplicationRecord
   belongs_to :account
   validates :phone,
             allow_blank: true,
-            format: { with: /\+[1-9]\d{1,14}\z/, message: "Número inválido" }
-
-
+            format: { with: /\+[1-9]\d{1,14}\z/ }
 
   has_many :deals, dependent: :destroy
   belongs_to :app, polymorphic: true, optional: true
 
-  def self.ransackable_attributes(auth_object = nil)
-    ["additional_attributes", "app_id", "app_type", "created_at", "custom_attributes", "email", "full_name", "id", "phone", "updated_at"]
+  def self.ransackable_attributes(_auth_object = nil)
+    %w[additional_attributes app_id app_type created_at custom_attributes email full_name id
+       phone updated_at]
```

Comment:
This change replaces a regular string array with the %w shorthand syntax. This is bad because it doesn’t follow the purpose of the PR.